pkg/informant/cgroup.go

```diff
@@ -78,62 +99,119 @@ func (s *CgroupState) setMemoryHigh(availableMemory uint64) error {
 
 	s.mgr.MemoryHighEvent.Consume()
 
-	if err := s.mgr.SetHighMem(newMemHigh); err != nil {
-		return fmt.Errorf("Error setting cgroup memory.high: %w", err)
+	memLimits := memoryLimits{
+		highBytes: newMemHigh,
+		maxBytes:  availableMemory,
+	}
+	if err := s.mgr.SetMemLimits(memLimits); err != nil {
+		return fmt.Errorf("Error setting cgroup memory limits: %w", err)
 	}
 
-	klog.Infof("Successfully set cgroup memory.high")
+	klog.Infof("Successfully set cgroup memory limits")
```

Comment:
I figured that the log line above was sufficient, because holding `s.updateMemLimitsLock` guarantees there's at most one call to `setMemoryLimits` ongoing at any time.


At the same time, I'm down for duplicating log information if it can be useful. Thoughts?

Reply:
it seems likely that putting the name of the group would be useful: I tend to think that context in logs is mostly useful because you can read them after the fact (or other people who have different understandings of the code and might not know about the mutex), and I think it's a useful property to be able to understand what's happening in a log without needing to read (many? any?) proximal log entries (particularly in systems with lots of concurrent action).

I guess it might not matter, but "what's the name of the cgroup" could be useful.

Reply:
Makes sense. I've added the cgroup name to a few of the logs.